Doc/library/html.parser.rst

```diff
@@ -134,7 +134,8 @@ implementations do nothing (except for :meth:`~HTMLParser.handle_startendtag`):
    argument is a list of ``(name, value)`` pairs containing the attributes found
    inside the tag's ``<>`` brackets.  The *name* will be translated to lower case,
    and quotes in the *value* have been removed, and character and entity references
-   have been replaced.
+   have been replaced. If a boolean attribute is encountered, the *value* for the
+   ``(name, value)`` attribute pair will be ``None``.
```

Comment:
In the HTML5 specification (https://html.spec.whatwg.org/multipage/syntax.html#attributes-2) they are called empty attributes.
```suggestion
   have been replaced. For empty attributes, *value* is ``None``.
```